src/client/components/region-banner-component/banner.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { Card, Container } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import 'client/components/region-banner-component/index.scss';
```

Comment:
It's good that you are using full path. But in this case, it's allowed to have a relative path `import './index.scss'.`